Bottom: 6d3bdc8ac4e3c317ef309adc8748d996cde94c97
Top:    154a91a3179cf44c752f15904df42dc68b9410fa
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-09-30 10:31:12 -0700

Refresh of file-clear-file

---

diff --git a/lldb/source/Host/common/File.cpp b/lldb/source/Host/common/File.cpp
index 368dc60759f..b79c00378c9 100644
--- a/lldb/source/Host/common/File.cpp
+++ b/lldb/source/Host/common/File.cpp
@@ -163,8 +163,7 @@ Status File::Close() {
 }
 
 FILE *File::TakeStreamAndClear() {
-  GetStream();
-  FILE *stream = m_stream;
+  FILE *stream = GetStream();
   m_stream = NULL;
   m_descriptor = kInvalidDescriptor;
   m_options = 0;
